exchanges/coinut/coinut_wrapper.go

```diff
@@ -141,8 +143,46 @@ func (c *COINUT) GetExchangeHistory(p pair.CurrencyPair, assetType string) ([]ex
 }
 
 // SubmitExchangeOrder submits a new order
-func (c *COINUT) SubmitExchangeOrder(p pair.CurrencyPair, side exchange.OrderSide, orderType exchange.OrderType, amount, price float64, clientID string) (int64, error) {
-	return 0, errors.New("not yet implemented")
+func (c *COINUT) SubmitExchangeOrder(p pair.CurrencyPair, side exchange.OrderSide, orderType exchange.OrderType, amount, price float64, clientID string) (string, error) {
+	var err error
+	var APIresponse interface{}
+	var response string
+	isBuyOrder := side == exchange.Buy
+	clientIDInt, err := strconv.ParseUint(clientID, 0, 32)
+	clientIDUint := uint32(clientIDInt)
+
+	if err != nil {
+		return "", err
+	}
+	// Need to get the ID of the currency sent
+	instruments, err := c.GetInstruments()
+	if err != nil {
+		return "", err
+	}
+
+	currencyArray := instruments.Instruments[p.Pair().String()]
+	currencyID := currencyArray[0].InstID
+
+	if orderType == exchange.Limit {
+		APIresponse, err = c.NewOrder(currencyID, amount, price, isBuyOrder, clientIDUint)
+	} else if orderType == exchange.Market {
+		APIresponse, err = c.NewOrder(currencyID, amount, 0, isBuyOrder, clientIDUint)
+	}
+	switch APIresponse.(type) {
+	case OrdersBase:
+		orderResult := APIresponse.(OrdersBase)
+		return fmt.Sprintf("%v", orderResult.OrderID), err
+	case OrderFilledResponse:
+		orderResult := APIresponse.(OrderFilledResponse)
+		return fmt.Sprintf("%v", orderResult.Order.OrderID), err
+	case OrderRejectResponse:
+		orderResult := APIresponse.(OrderRejectResponse)
+		// What should actually happen when its rejected?
```

Comment:
I don't see it needed either. Only thing would be to save an instance to verify why it failed. Other than that the ID can be thrown away.

Reply:
I'll log it in an error, rather than returning a happy result